resources/views/partials/tables/current_table.blade.php

```diff
@@ -31,23 +31,43 @@
                     @foreach ($activeVersions as $version)
                         <tr>
                             <th class="px-6 py-4 text-sm font-medium text-left text-gray-900 whitespace-nowrap">
-                                <a href="{{ $version->url }}" class="border-hover">{{ $version->major }} {{ $version->released_at->gt(now()) ? '(' . __('not released yet!') . ')' : '' }}</a>
+                                <a href="{{ $version->url }}" class="border-hover">{{ $version->major }} {{
+                                    $version->released_at->gt(now())
+                                    ? '(' . __('not released yet!') . ')'
+                                    : ''
+                                }}</a>
                             </th>
                             <td class="px-6 py-4 text-sm text-gray-500 whitespace-nowrap">
-                                {{ $version->released_at->gt(now()) ? $version->released_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')' : $version->released_at->translatedFormat(__('DateLongFormat')) }}
+                                {{ 
+                                    $version->released_at->gt(now())
+                                    ? $version->released_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')'
+                                    : $version->released_at->translatedFormat(__('DateLongFormat'))
+                                }}
                             </td>
                             <td class="px-6 py-4 text-sm text-gray-500 whitespace-nowrap">
                                 @if ($version->ends_bugfixes_at)
-                                    {{ $version->released_at->gt(now()) ? $version->ends_bugfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')' : $version->ends_bugfixes_at->translatedFormat(__('DateLongFormat')) }}
+                                    {{ 
+                                        $version->released_at->gt(now())
+                                        ? $version->ends_bugfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')'
+                                        : $version->ends_bugfixes_at->translatedFormat(__('DateLongFormat'))
+                                    }}
                                 @endif
                             </td>
                             <td class="px-6 py-4 text-sm text-gray-500 whitespace-nowrap">
                                 @if ($version->ends_securityfixes_at)
-                                    {{ $version->released_at->gt(now()) ? $version->ends_securityfixes_at->translatedFormat(__('DateShortFormat')) . ' (' . __('estimated') . ')' : $version->ends_securityfixes_at->translatedFormat(__('DateLongFormat')) }}
+                                    {{ 
+                                        $version->released_at->gt(now())
```

Comment:
Could you indent the second and third line of all of the instances where you changed the syntax like this?